imports/plugins/included/taxes-taxcloud/client/settings/taxcloud.js

```diff
@@ -20,8 +22,27 @@ Template.taxCloudSettings.helpers({
 AutoForm.hooks({
   "taxcloud-update-form": {
     onSuccess: function () {
-      return Alerts.toast(i18next.t("admin.taxSettings.shopTaxMethodsSaved"),
-        "success");
+      if (!TaxCodes.findOne({ taxCodeProvider: "taxes-taxcloud" })) {
+        Meteor.call("taxcloud/getTaxCodes", (err, res) => {
+          if (err) {
+            Alerts.toast(
+              `${i18next.t("admin.taxSettings.shopTaxMethodsFailed")} ${err}`, "error"
+            );
+          } else if (res && Array.isArray(res)) {
+            Alerts.toast(i18next.t("admin.taxSettings.shopTaxMethodsSaved"),
+              "success");
+            res.forEach((code) => {
+              Meteor.call("taxes/insertTaxCodes", Reaction.getShopId(), code, "taxes-taxcloud", (error) => {
+                if (error) {
+                  Alerts.toast(
```

Comment:
This is better, but the issue of toasting success and then toasting failure remains

Reply:
So should I scrape the toast of failure inside `onSuccess` hook? @spencern